siteimprove/admin/js/siteimprove-admin.js

```diff
@@ -37,7 +37,8 @@
 					$.post(
 						ajaxurl,
 						{
-							'action': 'siteimprove_request_token'
+							'action': 'siteimprove_request_token',
+							'_wpnonce': $( '#_wpnonce' ).val(),
```

Comment:
I suppose it's a fair assumption that the nonce already exists in a hidden input field due to some knowledge we have about the form.

Reply:
Yeah it already existed, just reusing it